libopenarc/arc.h

```diff
@@ -442,6 +443,20 @@ extern ARC_STAT arc_body __P((ARC_MESSAGE *msg, u_char *buf, size_t len));
 extern ARC_STAT arc_eom __P((ARC_MESSAGE *));
 
 /*
+**  ARC_CHECK_WHITELIST -- verify that all AS d= domains are present in the provided whitelist
+**
+**  Parameters:
+**  	msg -- ARC_MESSAGE object
+**    domain_whitelist -- an array of domain pointers
+**    whitelist_size -- number of domains in the whitelist
+**  Return value:
+**  	An ARC_STAT_* constant.
+**
+*/
+
+extern ARC_STAT arc_check_whitelist (ARC_MESSAGE *, char **, u_int);
```

Comment:
As we discussed on Wednesday, since this is a feature that's not actually part of the ARC specification, it should be implemented in openarc, not in libopenarc.